package.json

```
@@ -7,7 +7,7 @@
   "scripts": {
     "lint": "eslint .",
     "lint-fix": "eslint --fix .",
-    "test": "echo 'No tests implemented'",
+    "test": "jest --watch",
```

This really should be...
```suggestion
    "test": "jest",
```

if you need to watch you could do...
```
npm test -- --watch
```